tests/scale/virt-api.go

```diff
@@ -25,27 +27,24 @@ var _ = Describe("[sig-compute] virt-api scaling", decorators.SigCompute, func()
 	numberOfNodes := 0
 
 	setccs := func(ccs v12.CustomizeComponents) (oldcss v12.CustomizeComponents) {
-		originalKv := util.GetCurrentKv(virtClient)
-		kv, err := virtClient.KubeVirt(originalKv.Namespace).Get(context.Background(), originalKv.Name, v1.GetOptions{})
-		Expect(err).ToNot(HaveOccurred())
+		kv := util.GetCurrentKv(virtClient)
 		oldcss = kv.Spec.CustomizeComponents
-		kv.Spec.CustomizeComponents = ccs
+		patchPayload, err := patch.New(patch.WithReplace("/spec/customizeComponents", ccs)).GeneratePayload()
+		Expect(err).ToNot(HaveOccurred())
 		EventuallyWithOffset(1, func() error {
```

Comment:
Patch shouldn't fail we can replace the retry with
`Expect(err).ToNot(HaveOccurred())`

Reply:
Thanks, fixed.